CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Bob versions changelog
 - Update rust edition to 2021 (#484)
 
 #### Fixed
+- Remove unnecessary data clone (#506)
```

Comment:
I'd suggest to move this line to `Changed` section, because this is an improvement (not a bug)